screens/Request/RequestLayoutController.ts

```diff
@@ -20,7 +20,8 @@ import {
 } from '../../machines/bleShare/commonSelectors';
 import { RequestEvents } from '../../machines/bleShare/request/requestMachine';
 
-type RequestStackParamList = {
+export type RequestStackParamList = {
+  Request: undefined;
```

Comment:
Why does this have a type of `undefined` ? Esp if this is undefined like the rest of the entities in `RootStackParamList` in `index.ts`, can this be disabled on other platforms which might not support VC sharing?

Reply:
I referred to the previous UI code. Anyhow I'm doing a conditional rendering of this sharing VC options using "Platform" from react-native. In ios, it shouldn't behave as a verifier.